src/App.js

```diff
@@ -1,21 +1,103 @@
-import React, { useState, useEffect } from 'react';
-import PokemonList from './PokemonList';
-import axios from "axios";
-import './App.css';
+import React, { useState, useEffect } from "react";
+import PokemonList from "./PokemonList";
+import Axios from "axios";
+import "./App.css";
+import PaginationFunction from "./Pagination";
+import Spinner from 'react-bootstrap/Spinner'
+import Limit from "./selectLimit";
+import pokeBall from './Images/pokeBall.png'
```

Comment:
Common convention is to use word `assets` for any images or fonts

Reply:
added asset folder to directory location